src/test/java/tests/LoginTest.java

```diff
@@ -0,0 +1,50 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.BasePage;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class LoginTest extends BaseTest {
+    LoginPage page;
+    HomePage home;
+    BasePage base;
+
+    @BeforeMethod
+    public void loginPage(){
+        base = new BasePage(driver);
+        home = new HomePage(driver);
+        page = new LoginPage(driver);
+        home.openHomePage()
+                .toLoginPage();
+        String expected = "Kwidos";
+        String actual = driver.getTitle();
+        Assert.assertEquals(actual, expected);
```

Comment:
you can change 3 lines to one. Assert.assertEquals(driver.getTitle(), "Kwidos");